### ISSUE TITLE
Cannot receive data from Google Analytics

### ISSUE THREAD

**Issue by pkuyijia** [academicpages/academicpages.github.io#1561]
Hi, I try to use Google analytics to receive data for my personal website with tracking number G-XXXXXXXXXX

```
below is the current code for me:
# Analytics
analytics:
  provider               :  "google-universal" # false (default), "google", "google-universal", "custom"
  google:
    tracking_id          : G-XXXXXXXXXX
```

Is there something need to be changed, or there's some error in using google analytics now?

**Comment by dhancodes** [comment]
Try using custom option as provider and copy the code snippet from google analytics to ` _includes/analytics-providers/custom.html`

**Comment by bowei20222023** [comment]
I used that  _includes/analytics-providers/custom.htm but it seems not working as well.

**Comment by rjzupkoii** [comment]
This might be fixed now, please see #1183
